Ignore build.tcl or else it will create /usr/ports/devel/devel.tcl.
Tcllib bug [e334de2117].

Index: sak.tcl
--- sak.tcl.orig
+++ sak.tcl
@@ -227,6 +227,7 @@ proc ppackages {args} {
     foreach f $files {
 	# We ignore package indices and all files not in a module.
 
+	if {[string equal    build.tcl [file tail $f]]} {continue}
 	if {[string equal pkgIndex.tcl [file tail $f]]} {continue}
 	if {![regexp modules $f]}                       {continue}
 
